feat: add crn parser submodule (#2)
terraform-ibm-modules/terraform-ibm-common-utilities: README.md

https://terraform-ibm-modules.github.io/documentation/#/implementation-guidelines?id=module-names-and-descriptions
-->

> This repository serves as a repository to hold common Terraform utilities used in other modules in the terraform-ibm-modules organization.

Reviewer: "This repository serves as a library" maybe reads batter? - cc @SirSpidey

> This repository serves as a repository to hold common Terraform utilities used in other modules in the terraform-ibm-modules organization.

Reviewer 2: ```suggestion
Common Terraform utilities that are used in other modules in the terraform-ibm-modules GitHub organization.
```

I think you can shorten to get to the core info

> Each utility is placed under the [modules](./modules) directory so they may be individually referenced within a Terraform project, see the [Overview](#overview) section below for a list of available submodules.

Reviewer 2: ```suggestion
You can reference any utility in this repo from your Terraform project by pointing to the utility in the [modules](/modules) directory. For a list of available submodules, see the Submodules in the [Overview](#overview) section.
```
